aiohttp/web_reqrep.py

```diff
@@ -576,6 +590,20 @@ def _start_pre_check(self, request):
         else:
             return None
 
+    def _start_compression(self, request):
+        def start(coding):
+            self.headers[hdrs.CONTENT_ENCODING] = coding.name
+            self._resp_impl.add_compression_filter(coding.name)
```

Comment:
Good point. Perhaps better to skip compression if `coding` is `ContentCoding.identity`.
What is I really want is supporting `identity` encoding.